client/dvm/db/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+authors     = ["Darwinia Network <hello@darwinia.network>"]
+description = "Dvm database backend"
+edition     = "2018"
+homepage    = "https://darwinia.network/"
+license     = "GPL-3.0"
+name        = "dc-db"
+readme      = "README.md"
+repository  = "https://github.com/darwinia-network/darwinia-common/"
+version     = "1.3.0"
```

Comment:
We've bumped all the crate version in common library to `2.0.0`.

Related crates in this PR should be bumped too.